blockage/plugins.py

```diff
@@ -2,6 +2,22 @@
     import httplib
 except ImportError:
     import http.client as httplib
+
+try:
+    unicode = unicode
+except NameError:
+    # 'unicode' is undefined, must be Python 3
+    str = str
+    unicode = str
+    bytes = bytes
+    basestring = (str, bytes)
+else:
+    # 'unicode' exists, must be Python 2
+    str = str
+    unicode = unicode
+    bytes = str
+    basestring = basestring
```

Comment:
Well readable code is code, but code you don't need to read (because its not needed) is even better :smile: